gems/pending/openstack/openstack_handle/handled_list.rb

```diff
@@ -0,0 +1,118 @@
+module OpenstackHandle
+  module HandledList
+    def handled_list(collection_type, options = {})
```

Comment:
@ladas, the problem is that `"handled"` doesn't really describe what's going on.  

Change it to something like @matthewd's suggestion in order to describe what the method is actually handling.